nbconvert/_version_tools.py

```diff
@@ -0,0 +1,83 @@
+# import pep440 # cannot be imported as described below
+import re 
```

Comment:
It's @Carreau's pep440 to which you've committed, we can add a copyright notice if you like. Basically I'd just use it, but for the fact that it can't be built with setuptools.